backend/src/api/routes/company-route.ts

```diff
@@ -32,4 +59,87 @@ router.patch(
   }),
 );
 
+router.get(
+  '/okr',
+  validatePermissions([RoleType.Admin, RoleType.User]),
```

Comment:
Role `User` does not exist in the latest develop branch. Check if it is working. 
Users with roles `Mentor` and `Mentee` should be able to create OKR and then update OKR